puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-puppet-nightly-systest.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
```

Comment:
We know this job is currently broken because most (all?) modules don't allow puppet version 6 in their metadata. We should either pass in the parameter to skip the puppet version check to actually test it or disable the job. Right now I'm leaning to the latter since we don't really spend resources on it and I don't think anyone is actually checking this job.

Reply:
@ekohl i will disable this job.